gordian/repo.py

```diff
@@ -10,9 +10,9 @@
 BASE_URL = 'https://api.github.com'
 
 
-class Repo:
+class Repo(object):
```

Comment:
Why is this needed?

Reply:
Not needed it is there by default, something was not working and I was trying to change random things to "make it work", let me remove it.